### ISSUE TITLE
Indicate source of leaking async ops 

### ISSUE THREAD

**Issue by sgwilym** [denoland/deno#13644]
We recently started using Deno as the primary development runtime for a biggish pre-existing codebase. In the process of migrating tests I've become incredibly aware of how many `setTimeout` / `setInterval` operations were being called and not being cleaned up thanks to error messages like this:

```
AssertionError: Test case is leaking async ops.
Before:
  - dispatched: 0
  - completed: 0
After:
  - dispatched: 4043
  - completed: 4041
Ops:
  op_sleep:
    Before:
      - dispatched: 0
      - completed: 0
    After:
      - dispatched: 501
      - completed: 499
 ```

The only issue is that in a large codebase with dependencies this is really cryptic: there's no way to tell what the source of the `op_sleep`. So you have to spray and pray.

Would it be possible to show more information about leaking async ops so that it's easier to address them?

**Comment by lucacasonato** [comment]
Yes, I have an idea on how we can improve this. I'll get back to you on this soon.

**Comment by bartlomieju** [comment]
This has been implemented and shipped in Deno 1.19, starting with Deno 1.20 you need to pass `--trace-ops` flag to `deno test` to get the traces.
